src/app/modules/item/pages/item-forum/item-forum.component.ts

```diff
@@ -132,8 +153,30 @@ export class ItemForumComponent implements OnInit, OnChanges, OnDestroy {
     this.refresh$.next();
   }
 
+  toggleThread({
+    rowData,
+    itemId,
+    participantId
+  }:{
+      rowData: Thread & { isOpened: boolean, rowIndex: number },
+      itemId: string,
+      participantId: string,
+    }
+  ): void{
```

Comment:
and use a space here (may justify a eslint rule if it exists)
```suggestion
  ): void {
```